src/model/users/GetBatchUserImportErrorsResponse.ts

```diff
@@ -17,7 +17,7 @@ export interface GetBatchUserImportErrorsResponse {
     /**
      * The total number of failures for the specified user import
      */
-    'total_records': number;
+    'total_records': string;
```

Comment:
where these number changes to strings intentional 🤔 ?

Reply:
They are, this is not yet an issue, but in the future, it's possible that `total_records` will exceed the max integer for JS. So this is had to be updated in string format now. (coming from swagger spec `type: string,  format: int64` for this field). So the callers will have to parse the integer themselves. That said, I do think there are improvements that can be done here, but not in scope of this PR.